unraid_mcp/tools/storage.py

```
@@ -274,4 +274,56 @@ def format_bytes(bytes_value: int | None) -> str:
+        """
+        variables = {}
+        if importance:
+            variables["importance"] = importance.upper()
+
+        try:
+            logger.info(f"Executing archive_all_notifications tool: importance={importance}")
+            response_data = await make_graphql_request(query, variables)
+            return response_data.get("archiveAll", {})
+        except Exception as e:
+            logger.error(f"Error in archive_all_notifications: {e}", exc_info=True)
+            raise ToolError(f"Failed to archive notifications: {str(e)}") from e
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)